docker/airflow/entrypoint.sh

```diff
@@ -34,6 +34,21 @@ function header() {
 if [ "$1" == help ] || [ "$1" == --help ]; then help_text && exit 0; fi
 sleep 0.1;  # The $COLUMNS variable takes a moment to populate
 
+# Reformat Airflow connections that use https
+header "MODIFYING ENVIRONMENT"
+# Loop through environment variables, relying on naming conventions
+while IFS="="; read var_name var_value; do
```

Comment:
Just because I'm sure to come back to this in 6 months and forget how it all works 😅 
```suggestion
# Loop through environment variables, relying on naming conventions.
# Bash loops with pipes occur in a subprocess, so we need to do some special
# subprocess manipulation via <(...) syntax in order to allow the `export` calls
# to propagate to the outer shell.
# See: https://unix.stackexchange.com/a/402752
while IFS="="; read var_name var_value; do
```